docs/content/guides/dagster/enriching-with-software-defined-assets.mdx

```diff
@@ -0,0 +1,407 @@
+---
+title: Upgrading To Software-Defined Assets | Dagster
+description: Understand when, why, and how to use software-defined assets (SDAs) in Dagster, as well as how SDAs work with other core Dagster concepts.
+---
+
+# Upgrading To Software-Defined Assets
+
+If you want to understand when, why, and how to use software-defined assets in Dagster, this guide is for you. We'll also demonstrate what some common Dagster jobs look like before and after using software-defined assets.
```

Comment:
Based on the first sentence, I think it's a little difficult to know what to expect for how the content in here would be different than the content in - say - the SDA concepts page. I think it's worth trying to target it a little bit more specifically. I can brainstorm options with you if it would be helpful.

Reply:
> If you're familiar with ops and graphs and want to understand when, why, and how to use software-defined assets in Dagster, this guide is for you. We'll also demonstrate what some common Dagster jobs look like before and after using software-defined assets.